src/App/Service/Github/Search.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Console\App\Service\Github;
+
+use Github\Api\AbstractApi;
+use Github\Api\AcceptHeaderTrait;
+
+/**
+ * Implement the Search API. To override the default class that not support per_page and page parameters,
+ */
+class Search extends AbstractApi
```

Comment:
Why not extends the Search class for KNPLabs package for just overriding the good method ?

(And contribute to KNPLabs package, of course).

Reply:
So.... I' ve adjusted all method of the class it makes sense to override all the class.
And about contrib to the native package will be a bit more tricky. The maintainer removes the getters for per_page and page param in AbstractApi class. I will have to discuss with him before.